### PR TITLE
Enable TravisCI

### PR DESCRIPTION
**Pull request by ncloudioj**
r? @rlr

### CODE REVIEW

**Comment by rlr** [comment]
no luck eh?

**Comment by ncloudioj** [comment]
Hold on tight! Still isolating the dependent libs here.

I think I can get it work soon. Stay tuned!

File: test/disco_tests/test_ip_counter.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import unittest
 from mock import Mock
-from infernyx.rules import parse_ip_clicks
-from infernyx.rules import filter_clicks
+from infernyx.rule_helpers import parse_ip_clicks, filter_clicks
```
Comment (rlr) [inline test/disco_tests/test_ip_counter.py:3]: should you remove rules.py now? or delete some of the duplication?

Comment (rlr) [reply test/disco_tests/test_ip_counter.py:3]: oh nevermind. github wasn't showing me that diff by default! R+

**Review by rlr — APPROVED** [review]
yay!

**Pull request merged**